src/LogViewer/chunk-worker.js

```diff
@@ -10,6 +10,22 @@ const LINE_CHUNK = 1000;
 const DECODER = new TextDecoder('utf-8');
 // Setting a hard limit on lines since browser have trouble with heights starting at around 16.7 million pixels and up
 const CHUNK_LIMIT = 883000 / LINE_CHUNK;
+// HTML Escape helper utility
```

Comment:
I know that this code was taken pretty much as-is from that article I sent you, but I'm going to make some changes and explain.